app/database2.js

```diff
@@ -0,0 +1,58 @@
+let express = require("express");
+let { Pool } = require("pg");
+let env = require("E:/starter2/dbenv.json");
+const argon2 = require('argon2');
+
+let hostname = "localhost";
+let port = 3000;
+let app = express();
+
+app.use(express.json());
+
+let pool = new Pool(env);
+pool.connect().then(() => {
+    console.log("Connected to database");
+});
+
+app.post('/login', async (req, res) => {
+    let { username, password } = {username:'user11', password:'12345'};
+    /*
+    let body = req.body;
+    let { username, password } = body;  used when data is sent from frontend
+    */
+  
+    try {
+      let queryResult = await pool.query('SELECT * FROM userdata WHERE username = $1', [username]);
+      let user = queryResult.rows[0];
+      
+
+      const hashpassword = await argon2.hash(password);
+      console.log(hashpassword);
+      if (!user) {
+        await pool.query('INSERT INTO userdata (username, password, loggedin) VALUES ($1, $2, $3)', [
```

Comment:
I see you are creating new accounts here. That makes sense for now, but we will eventually want a separate "Create Account" system. That way, you don't accidentally create a new account if you misspell your username

Reply:
The backend is only for testing, the eventual backend is developed by Matt